modules/nw-osp-pod-creating-ipv6.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-creating-ipv6_{context}"]
+= Create pods that have IPv6 connectivity on {rh-openstack}
+
+After you enable IPv6 connectivty for pods and add it to them, create pods that have secondary IPv6 connections. 
```

Comment:
Purely a suggestion would a Prerequisites section work here?

Reply:
Totally would, but given time, in-shortdesc explanation, and the unlikeliness that these modules won't be chained to each other ever, I'm going to call this an enhancement. :grimacing: